rebar.config

```diff
@@ -1,7 +1,7 @@
 {erl_opts, [debug_info, {parse_transform, lager_transform},
             {lager_extra_sinks, [epoch_mining, epoch_pow_cuckoo,
                                  epoch_metrics]}]}.
-{deps, [{lager, ".*", {git, "https://github.com/erlang-lager/lager.git", {tag, "3.5.1"}}},
+{deps, [{lager, ".*", {git, "https://github.com/aeternity/lager.git", {ref, "5abb0c1"}}},
```

Comment:
Does this close a Pivotal ticket?

Reply:
Yes, https://www.pivotaltracker.com/story/show/153772726

Reply:
https://www.pivotaltracker.com/n/projects/2124891/stories/153772726